src/app-layout/visual-refresh/trigger-button.scss

```diff
@@ -51,4 +52,24 @@ handleSplitPanelMaxWidth function in the context.
       background: awsui.$color-background-layout-toggle-selected-active;
     }
   }
+
+  &.badge {
+    clip-path: path(
+      'M38.0215 11.3161C37.1068 11.7545 36.0821 12 35 12C31.134 12 28 8.86599 28 5C28 3.91794 28.2455 2.89322 28.6839 1.97851C26.0575 0.710652 23.1118 0 20 0C8.9543 0 0 8.9543 0 20C0 31.0457 8.9543 40 20 40C31.0457 40 40 31.0457 40 20C40 16.8882 39.2893 13.9425 38.0215 11.3161Z'
```

Comment:
Do we still keep this clip-path? Does Safari like it?

Reply:
I checked this one on Safari - its fine because its on the button element and not directly on the svg. Still a mystery why Safari didn't like the svg clip-path specifically, want to do some more digging.